pyartifactory/objects/artifact.py

```diff
@@ -121,11 +126,7 @@ def deploy(
                     raise ArtifactoryError from error
             else:
                 with local_file.open("rb") as stream:
-                    properties_param_str = ""
-                    if properties is not None:
-                        properties_param_str = self._format_properties(properties)
-                    route = ";".join(s for s in [artifact_folder.as_posix(), properties_param_str] if s)
-                    self._put(route, data=stream)
+                    self._put(route=route, headers=headers, data=stream)
```

Comment:
I believe you're missing the following line before the _put method:

```
headers["X-Checksum-Deploy"] = "false"
```

Reply:
Done